src/modules/tracker/components/TrackerList.vue

```diff
@@ -131,27 +127,9 @@ function getBillable(x) {
               </li>
             </ul>
           </td>
-          <td><input class="edit" type="text" v-model="log.start_time" /></td>
-          <td><input class="edit" type="text" v-model="log.end_time" /></td>
-
+          <td><input class="edit" type="text" v-model="log.start_time" @focusout="editLogs(log.activity_name, userId, log.project_id, log.billable, log.start_time, log.end_time, log.id)" @keyup.enter="editLogs(log.activity_name, userId, log.project_id, log.billable, log.start_time, log.end_time, log.id)" /></td>
+          <td><input class="edit" type="text" v-model="log.end_time" @focusout="editLogs(log.activity_name, userId, log.project_id, log.billable, log.start_time, log.end_time, log.id)" @keyup.enter="editLogs(log.activity_name, userId, log.project_id, log.billable, log.start_time, log.end_time, log.id)"/></td>
```

Comment:
Instead of passing multiple arguments to "editLogs" function, just pass "log" object and destrcture in the "editLogs" function